aten/src/ATen/native/cuda/BinaryOpsKernel.cu

```diff
@@ -13,12 +13,18 @@
 namespace at { namespace native {
 
 void add_kernel_cuda(TensorIterator& iter, Scalar alpha_scalar) {
```

Comment:
We need a strategy for dealing with "alpha_scalar" with bools. (For CPU too). Require that it's 1?

Reply:
Can we enforce such a requirement? This will prevent `-` be interpreted as `||` for bool.